### ISSUE TITLE
webcam solution

### ISSUE THREAD

**Issue by milestonezero** [FongYoong/u2net-tfjs#1]
Thanks for the amazing work, is there possibility we can do webcam because I have very light weight model and can be run at real time.

**Comment by FongYoong** [comment]
Yes, it's definitely doable, but I can't help make a prototype unless I have access to a sufficiently fast model.
